### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by PhilippMeissner** [AlexKhymenko/ngx-permissions#196]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## Current behavior

When running our tests with jest we receive the error `SyntaxError: Cannot use import statement outside a module`

 ```
C:\path-to-project\node_modules\ngx-permissions\fesm2015\ngx-permissions.mjs:1
({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import * as i0 from '@angular/core';
^^^^^^
 SyntaxError: Cannot use import statement outside a module
```

## Expected behavior
The import should just work as is.

Basically just create a new angular project with the given version below, add jest and jest-preset-angular. Additionally add and import the NgxPermissionsModule and run the tests.
## Environment

<pre><code>
Angular version: 12.2.3
ngx-permissions version: 13.0.1
jest version: 27.5.1
jest-preset-angular: 11.1.1
Node 12.16.3
</code></pre>
